src/odemis/acq/stream/_static.py

```diff
@@ -479,6 +479,10 @@ def _project2Polar(self, pos):
                     # Simple version: remove the background value
                     data0 = angleres.ARBackgroundSubtract(data)
                 else:
+                    # image shape of image and bg image might be not the same due to resizing
```

Comment:
Could we first subtract the background, and then rescale (if needed)? That would avoid calling twice `rescale_hq()`